src/metric/populate_test.go

```diff
@@ -21,101 +27,107 @@ func parseTime(raw string) time.Time {
-			Name: "newrelic-infra-rz225",
-			Type: "k8s:test-cluster:kube-system:pod",
+		Metadata: &integration.EntityMetadata{
+			Name:      "newrelic-infra-rz225",
+			Namespace: "k8s:test-cluster:kube-system:pod",
+			IDAttrs:   integration.IDAttributes{},
 		},
-		Metrics: []sdkMetric.MetricSet{
+		Metrics: []*metric.Set{
 			{
-				"entityName":                     "k8s:test-cluster:kube-system:pod:newrelic-infra-rz225",
-				"event_type":                     "K8sPodSample",
-				"net.rxBytesPerSecond":           0., // 106175985, but is RATE
-				"net.txBytesPerSecond":           0., // 35714359, but is RATE
-				"net.errorsPerSecond":            0.,
-				"createdAt":                      parseTime("2018-02-14T16:26:33Z").Unix(),
-				"startTime":                      parseTime("2018-02-14T16:26:33Z").Unix(),
-				"createdKind":                    "DaemonSet",
-				"createdBy":                      "newrelic-infra",
-				"nodeIP":                         "192.168.99.100",
-				"namespace":                      "kube-system",
-				"namespaceName":                  "kube-system",
-				"nodeName":                       "minikube",
-				"podName":                        "newrelic-infra-rz225",
-				"isReady":                        1,
-				"status":                         "Running",
-				"isScheduled":                    1,
-				"label.controller-revision-hash": "3887482659",
-				"label.name":                     "newrelic-infra",
-				"label.pod-template-generation":  "1",
-				"displayName":                    "newrelic-infra-rz225", // From manipulator
-				"clusterName":                    "test-cluster",         // From manipulator
+				Metrics: map[string]interface{}{
+					"event_type":                     "K8sPodSample",
+					"net.rxBytesPerSecond":           0., // 106175985, but is RATE
+					"net.txBytesPerSecond":           0., // 35714359, but is RATE
+					"net.errorsPerSecond":            0.,
+					"createdAt":                      float64(parseTime("2018-02-14T16:26:33Z").Unix()),
+					"startTime":                      float64(parseTime("2018-02-14T16:26:33Z").Unix()),
+					"createdKind":                    "DaemonSet",
+					"createdBy":                      "newrelic-infra",
+					"nodeIP":                         "192.168.99.100",
+					"namespace":                      "kube-system",
+					"namespaceName":                  "kube-system",
+					"nodeName":                       "minikube",
+					"podName":                        "newrelic-infra-rz225",
+					"isReady":                        float64(1),
+					"status":                         "Running",
+					"isScheduled":                    float64(1),
+					"label.controller-revision-hash": "3887482659",
+					"label.name":                     "newrelic-infra",
+					"label.pod-template-generation":  "1",
+					"displayName":                    "newrelic-infra-rz225", // From manipulator
+					"clusterName":                    "test-cluster",         // From manipulator
```

Comment:
The manipulators has been removed?
```suggestion
					"displayName":                    "newrelic-infra-rz225",
					"clusterName":                    "test-cluster",
```

Reply:
Done!